pkg/miniogw/integration_test.go

```diff
@@ -79,7 +79,9 @@ func TestUploadDownload(t *testing.T) {
 
 	// keys
 	uplinkCfg.Client.APIKey = "apiKey"
-	uplinkCfg.Enc.Key = "encKey"
+	const humanReadableEncKey = "encKey"
+	encKey := new(storj.Key)
+	copy(encKey[:], humanReadableEncKey)
```

Comment:
Shouldn't we use `storj.NewKey` here too?

Reply:
yes, it should, I don't know why `NewKey` wasn't used, maybe it didn't exist at that time or basically missed with one of the upstream updates.